README.md

```diff
@@ -182,7 +188,11 @@ wget https://raw.githubusercontent.com/wwood/CoverM/refs/heads/main/demo/genome_
 wget https://raw.githubusercontent.com/wwood/CoverM/refs/heads/main/demo/genome_8.fna
 ```
 
-Run CoverM
+Next, we can actually run CoverM.
+We use `--coupled` to specify the paired-end reads, and `--genome-fasta-files` to specify the genomes.
+We also specify the number of threads to use with `-t`, and the coverage metrics we want to calculate with `-m`.
+We will calculate the mean coverage, relative abundance, and covered fraction for each genome in the sample.
+The output will be written to `output_coverm.tsv`.
```

Comment:
I've asked Nanda to work through the demo, to give it another perspective.

Actually I think also printing what the ouput file looks like (as a markdown table) would be helpful.